modules/5_install/install.tf

```diff
@@ -41,7 +41,7 @@ locals {
     cluster_domain        = local.cluster_domain
     cluster_id            = var.cluster_id
     bastion_ip            = var.bastion_vip != "" ? var.bastion_vip : var.bastion_ip[0]
-    bastion_name          = var.bastion_vip != "" ? "${var.cluster_id}-bastion" : "${var.cluster_id}-bastion-0"
+    bastion_name          = var.bastion_vip != "" ? "${var.name_prefix}-bastion" : "${var.name_prefix}-bastion-0"
```

Comment:
This is not needed if we are not changing the hostname and sticking with only resource name change for adding zone.

Reply:
Closing this comment as we decided to have zone name in hostname for the nodes.